doc/users/next_whats_new/inset_axes_improvements.rst

```diff
@@ -0,0 +1,6 @@
+Axes.inset_axes Flexibility
+---------------------------
+
+`matplotlib.axes.Axes.inset_axes` now accepts the ``projection``, ``polar``
+and ``axes_class`` keywords, so that subclasses of `matplotlib.axes.Axes` may
```

Comment:
Argument names should use italics.
```suggestion
`matplotlib.axes.Axes.inset_axes` now accepts the *projection*, *polar* and
*axes_class* keyword arguments, so that subclasses of `matplotlib.axes.Axes` may
```